src/components/auth/LoginForm/index.jsx

```diff
@@ -1,38 +1,42 @@
 import {useState} from "react";
 import Logo from "components/main/Logo";
 import Button from "components/main/Button";
+import useAuth from "hooks/auth/useAuth";
 
 export const LoginForm = () => {
-    const [userInputValue,setUserInputValue] = useState('')
-    const [passwordInputValue,setPasswordInputValue] = useState('')
+    const {login} = useAuth()
+    const [userInputValue, setUserInputValue] = useState('jatmolina15@gmail.com')
```

Comment:
remove default values from username and password, keep just for development purposes